### ISSUE TITLE
Does not work without jquery in React-Native

### ISSUE THREAD

**Issue by IanVS** [AceMetrix/jquery-deparam#23]
React-native has a `require` function, but it is not the same as that in node.js.  One critical difference is that it does not support dynamic requires, and does not obey the try/catch block.  All requires are done up-front upon static analysis so that the assets can all be packaged into a bundle.

At any rate, in my React-Native app, I do not use jQuery.  I tried to use `jquery-deparam`, but with no luck.  React-Native fails to bundle, because it cannot `require(jquery)`.  

Do you think it would be possible to make the jQuery dependency injection more explicit?  Perhaps with something like:

```js
import addDeparam from 'jquery-deparam';
import $ from 'jquery';

const $ = addDeparam($);  // Adds `deparam` method to jQuery
const deparam = addDeparam(); // define `deparam` function to use standalone
```

**Comment by badunk** [comment]
hm, I would imagine this problem would exist for other users of UMD and react-native.  Basically, it your environment goes into the node js case, when it shouldn't.

**Comment by IanVS** [comment]
Yes, I verified that in my case, removing the first `if` block (which checks for `require`) works correctly.
